gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notifications-list.component.html

```diff
@@ -44,6 +44,25 @@ <h1>Notifications</h1>
       </td>
     </ng-container>
 
+    <ng-container matColumnDef="actions">
+      <th mat-header-cell *matHeaderCellDef id="actions" width="1%"></th>
+      <td mat-cell *matCellDef="let element">
+        <div class="endpoint-list__table__actions">
+          <ng-container *ngIf="element.id">
+            <button
+              *gioPermission="{ anyOf: ['api-definition-u'] }"
```

Comment:
For permission management. And even other business rules. We need to look at what was done before to avoid regression.
For example, for this button the previous permissions are : 
https://github.com/gravitee-io/gravitee-api-management/blob/f46af8e79d0aac956b447356ca9b3ad0e25be8ba/gravitee-apim-console-webui/src/components/notifications/notifications.html#L115

Reply:
so as we create first notification only for API : we need to use : api-notification-c ;)

Reply:
Updated during a call